WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -143,16 +144,16 @@ def getAttributesForJobList(self, jobIDList, attrList=None):
         jobID = retValues[0]
         jobDict = {}
         jobDict['JobID'] = jobID
-        attrValues = retValues[1:]
-        for i in xrange(len(attr_tmp_list)):
+        for name, value in zip(attr_tmp_list, retValues[1:]):
```

Comment:
I guess a small comment to explain what is going on would not harm, because I have no clue ! :D